### ISSUE TITLE
about the cls_loss and reg_loss compute

### ISSUE THREAD

**Issue by muzi2045** [jhultman/PV-RCNN#20]
this line code in proposal.py 
`loss = cls_loss + self.cfg.TRAIN.LAMBDA * reg_loss`
https://github.com/jhultman/PV-RCNN/blob/master/pvrcnn/detector/proposal.py#L139

why the total loss = cls_loss + 0.1*reg_loss ? which the cfg.TRAIN.LAMBDA usually set to 0.1
that not make sense...
